### ISSUE TITLE
You can enable pro features without a license

### ISSUE THREAD

**Issue by RobinArlinghaus** [alin23/Lunar#449]
# Issue details
- Mac device where Lunar is installed (Macbook Pro 2019, iMac M1, Mac Mini M1, Hackintosh etc.):
    MacBook Pro 2021

- Monitor model (LG UltraFine 5K, DELL P2715Q etc.):
    Internal Display

- Monitor connection to the Mac device (HDMI-to-USB-C, USB-C-to-USB-C, miniDisplayPort-to-DisplayPort etc.):
    none

- Using an USB Docking Station or Hub:
    none

- Lunar mode used (check it in the top-right corner of the Lunar interface)
    Manual

- Tried Lunar Diagnostics?
> 1. Click on **Open Lunar Diagnostics** in the Lunar menu
> 2. Follow through the steps when asked

![Lunar Diagnostics menu item](https://files.lunar.fyi/diagnostics-menu-item.png)

    no

# Issue description:
I somehow managed to enable pro features in Lunar. Im not entirely sure how it happened, but i was able to do it twice. When I installed Lunar, I disabled internet access for the app, then i clicked on "get Pro" by accident. Because I couldnt close the dialog that showed up, I killed the process with htop. When I restarted Lunar, I had all the Pro features and it said "Pro" in the preferences. I dont know if thats right, but the option to activate a license is still there.

**Comment by alin23** [comment]
Huh, interesting. Can you share how you disabled internet access for Lunar?

**Comment by RobinArlinghaus** [comment]
I downloaded it today for the first time, dont have a license or anything. Not sure what you mean with trial.

**Comment by alin23** [comment]
By the way, if you're worried about privacy, here's the policy: [Lunar - Privacy Policy](https://lunar.fyi/privacy)

TLDR: I don't care about personal data or how you use the app. I collect only generic monitor data and make it available for everyone in case others want to see what monitors support DDC: [The Monitor Database](https://db.lunar.fyi)
